pontoon/projects/views.py

```diff
@@ -62,6 +71,90 @@ def ajax_info(request, slug):
     })
 
 
+@permission_required_or_403('base.can_manage_project')
+@transaction.atomic
+@require_AJAX
+def ajax_notifications(request, slug):
+    """Notifications tab."""
+    project = get_object_or_404(Project.objects.available(), slug=slug)
+    available_locales = project.locales.order_by('name')
+
+    # Send notifications
+    if request.method == 'POST':
+        message = request.POST.get('message', None)
+        selected_ids = request.POST.get('selected_locales')
```

Comment:
I think we could use a django form here to give validation the better structure.